.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-- '2.7'
+- '3.5'
```

Comment:
@RamanjaneyuluIdavalapati Is it possible to support more than one version of python? i.e. Can we support both 2 and 3 at the same time? Just asking, it is okay if this module is 3 only.

Reply:
@prashanthellina 
- wordvecspace service depends on Kwikapi. But Kwikapi only works on Python3. So we cannot support wordvecspace service for Python2.
- We can able to support wordvecspace interact and convert for both Python 2 and Python 3.
But here is the problem.
- We are using xrange function in normalize_vectors function of wordvecspace. In Python3 range was removed and xrange is renamed to the range. So we must use range for Python3.
- If we use the range for Python3 then if the user works on python2 then actual range function will be called and it takes the more memory to form list (because of parallelization).